src/utils/sidebar.js

```diff
@@ -22,11 +18,18 @@ const sidebar = require('../../public/static/docs/sidebar.json')
   }
 */
 
+const startCase = require('lodash.startcase')
+
+/* Base sidebar structure */
```

Comment:
I think comments like this are not informative and don't provide much value

Reply:
OK. We've discussed comments before and I'm not trying to add complete file documentation. But we have some ad hoc patterns in the docs engine and I comment things that took me some time to figure out. In this case, this signals that the structure in the long file comment above is NOT the same as the one in sidebar.json, but rather that it's just based on the JSON file.

Reply:
I also separated it from the previous `require` because this one is less of an "import" and more of a constant definition.

Reply:
I still think that code is pretty much self-descriptive and comments like this do not provide much value )

Reply:
No worries, I will strip down as many comments as I can when this PR is no longer draft.